modules/creating-microshift-development-environment.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="creating-microshift-development-environment"]
+= Creating the MicroShift development environment
+:context: creating-microshift-development-environment
+
+Use the following procedure to set up your MicroShift development environment. 
+
+.Procedure 
+
+. Download {op-system-first} 8.7 or greater from the link:https://developers.redhat.com/products/rhel/download[Download Red Hat Enterprise Linux] page. 
++
+[NOTE]
+====
+{op-system} 9.0 is currently unsupported for MicroShift deployments.
+====
+
+. Create a {op-system} 8 virtual machine with the following settings: 
+.. 2 cores
+.. 4096 MB of RAM
+.. 50 GB of storage 
++
+[NOTE]
+====
+Visual Studio Code might consume up to 2 GB of RAM. For running the IDE on the development virtual machine, it is recommended to allocate at least 6144 MB of RAM in total. 
+====
+
+. Move the ISO image to the `/var/lib/libvirt/images` directory. 
+
+. Enter the following command to create the virtual machine: 
```

Comment:
```suggestion
. Enter the following commands to create the virtual machine: 
```

This step on line 33 is the implementation of the instruction for the step on line 21. Do we want to move the step on line 31 up above (the user needs to do that before creating the VM with this command) and then merge the other 2 steps into 1 thing? Or maybe the stuff about the VM parameters on lines 21-24 should move above the procedure entirely and be rephrased as something like "To experiment with MicroShift, you will need a VM with ..."